Update agent release page (#6695)
DataDog/integrations-core: docs/developer/process/agent-release.md

## Setup

> Configure and add the following to your [configuration](../ddev/configuration.md):

Reviewer: ```suggestion
Ensure that you have configured the following:
```

> Configure your the following and add to your [configuration](../ddev/configuration.md):

Reviewer 2: ```suggestion
Configure and add the following to your [configuration](../ddev/configuration.md):
```

- [GitHub](../ddev/configuration.md#github) credentials
- [Trello](../ddev/configuration.md#trello) credentials

> - [Trello team mappings](../ddev/configuration.md#item-assignments)

Reviewer: ```suggestion
- [Trello team mappings](../ddev/configuration.md#card-assignment)
```

> - Add the `trello_users_$team` table to [automatically assign test cards](#create-items) to team members.

Reviewer 2: > `trello_users_$team` table

Should we add/move instructions on how to set these up to the Trello config docs?

Currently the "create items" section reads:

> You must assign each item to a team member after creation and ensure no one is assigned to a change that they authored. [...]

But I think it's fair to make the team mapping a required part of the setup, and just not mention those manual steps anymore.

Then we can say:

```suggestion
- [Trello teams mapping](../dev/configuration.md#trello-teams) (for assigning card items automatically)
```

Don't know if I'm being clear, let me know if this needs clarifying.

Author: That makes sense, I can move the trello team mapping as another section in trello configuration.

1. Make a pull request to release [any new integrations](integration-release.md#new-integrations), then merge it and pull `master`
1. Make a pull request to release [all changed integrations](integration-release.md#bulk-releases), then merge it and pull `master`

!!! important

>     Update [PyPI](integration-release.md#PyPI) if you released `datadog_checks_base` or `datadog_checks_dev`.

Reviewer 2: (Nit) Newline separation with the next h3 title

```suggestion
    Update [PyPI](integration-release.md#PyPI) if you released `datadog_checks_base` or `datadog_checks_dev`.

```

would select all commits that were merged between the Git references.

The command will display each change and prompt you to assign a team or skip. Purely documentation changes are automatically skipped.

> Items are automatically assigned if `$trello_users_$team` table is [configured](../ddev/configuration.md#item-assignments).

Reviewer: ```suggestion

Cards are automatically assigned if `$trello_users_$team` table is [configured](../ddev/configuration.md#card-assignment).
```

### Communication

> Update the `#agent-release-sync` channel with a daily status.

Reviewer: ```suggestion
```

ddev agent integrations
```

See more options for [`ddev agent changelog`](../ddev/cli.md#changelog) and [`ddev agent integrations`](../ddev/cli.md#integrations).

> Update the contents of the [`AGENT_CHANGELOG`][agent-changelog] and [`AGENT_INTEGRATIONS`][agent-integrations] files. 

Reviewer: Do like:

1. run A to update X
1. run B to update Y

Author: updated

ddev agent integrations
```

> Update the [`AGENT_CHANGELOG`][agent-changelog] and [`AGENT_INTEGRATIONS`][agent-integrations] files. Create a pull request and wait for approval before merging.

Reviewer 2: Yes, these commands just output stuff to the console so good point to instruct to actually update the files.

Nit copy suggestion to make this action item even more explicit:

```suggestion
Update the [`AGENT_CHANGELOG`][agent-changelog] and [`AGENT_INTEGRATIONS`][agent-integrations] files with the outputs of these commands. Create a pull request and wait for approval before merging.
```

ddev agent integrations
```

> Update the [`AGENT_CHANGELOG`][agent-changelog] and [`AGENT_INTEGRATIONS`][agent-integrations] files with the outputs of these commands. Create a pull request and wait for approval before merging.

Reviewer: Link commands

Author: The `ddev agent changelog/integrations` commands?

Author: updated!
